scripts/subscription-send.md

```diff
@@ -0,0 +1,41 @@
+# Subscription-Send script
+<!-- TOC -->
+* [Subscription-Send script](#subscription-send-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion/QuickGuide:](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The Subscription-Send script is designed to send email notifications related to subscriptions in DSpace. Users can
+subscribe to specific communities, collections, or items.
+
+## When we use it?
+
+The Subscription-Send script is used to send periodic email updates based on the user's subscription frequency.
```

Comment:
Please add a link to official docs.

"For more information see the [Email Subscriptions](https://wiki.lyrasis.org/display/DSDOC7x/Email+Subscriptions) documentation."